<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDlg</class>
 <widget class="QDialog" name="LoginDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>313</width>
    <height>234</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>LoginDlg</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	border-radius:3px;
	border:1px groove gray;
}

QToolButton {
	border:0;
}
QToolButton#closeBtn{
	background-image: url(:/images/close.png);
}

QToolButton#closeBtn:hover {
	background-image: url(:/images/close_h.png);
}

QToolButton#closeBtn:pressed {	
	background-image: url(:/images/close_p.png);
}


QLineEdit {
	color:rgb(127,0,63);
	border:2px groove gray;
	border-radius:6px;
	selection-color: white;
    selection-background-color: rgb(191, 31, 127);
}

QPushButton {
    color: white;
    font: bold 10pt;
    border-image: url(:/images/button.png) 16;
    border-width: 16px;
    padding: -16px 0px;
    min-height: 32px;
    min-width: 60px;
}

QPushButton:hover{
	border-image:url(:/images/button_h.png) 16;
}
QPushButton:pressed{
	border-image:url(:/images/button_p.png) 16;
}
QLabel {
	color:gray;
	
}
QCheckBox {
	color:gray;
}


</string>
  </property>
  <widget class="QToolButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>293</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="MyImages.qrc">:/images/headimg.png</pixmap>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>240</width>
     <height>124</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>密码：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="passwordLineEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="userNameLineEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>账号：</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="checkBox">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>记住账号</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="loginBtn">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>登录</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="cancelBtn">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>label_3</zorder>
  <zorder>closeBtn</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>userNameLineEdit</tabstop>
  <tabstop>passwordLineEdit</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>loginBtn</tabstop>
  <tabstop>cancelBtn</tabstop>
  <tabstop>closeBtn</tabstop>
 </tabstops>
 <resources>
  <include location="MyImages.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeBtn</sender>
   <signal>clicked()</signal>
   <receiver>LoginDlg</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>9</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>LoginDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
